src/utilityScriptSerializers.ts

```diff
@@ -135,3 +135,47 @@ export function isPrimitiveValue(value: any): boolean {
       return false;
   }
 }
+
+const autoClosingTags = new Set(['AREA', 'BASE', 'BR', 'COL', 'COMMAND', 'EMBED', 'HR', 'IMG', 'INPUT', 'KEYGEN', 'LINK', 'MENUITEM', 'META', 'PARAM', 'SOURCE', 'TRACK', 'WBR']);
+const booleanAttributes = new Set(['checked', 'selected', 'disabled', 'readonly', 'multiple']);
+
+function oneLine(s: string): string {
+  return s.replace(/\n/g, '↵').replace(/\t/g, '⇆');
+}
+
+export function previewNode(node: Node): string {
```

Comment:
utilityScriptSerializers.ts is used from both Node and Browser. Your function is only used from the browser, so it should be elsewhere?

Reply:
Fixed.